src/optimum/intel/neural_compressor/config.py

```diff
@@ -93,3 +92,28 @@ def from_pretrained(
 
         config = cls(config_file, **config_kwargs)
         return config
+
+
+class DeployIncConfig(IncConfig):
+
+    def __init__(self, config_path: str):
+        """
+        Args:
+            config_path (:obj:`str`):
+                Path to the YAML configuration file used to control the tuning behavior.
+        Returns:
+            config: DeployIncConfig object.
+        """
+
+        self.path = config_path
+        self.config = self._read_config()
+        self.usr_cfg = self.config
+
+    def _read_config(self):
+        with open(self.path, 'r') as f:
+            try:
+                config = yaml.load(f, Loader=yaml.Loader)
+            except yaml.YAMLError as exc:
+                print(exc)
```

Comment:
Why you are not using the logger as in run_glue?

Reply:
Good catch, thanks !